tests/storage/disk_preallocation/test_disk_preallocation.py

```
@@ -0,0 +1,62 @@
+    registry_dv_with_preallocation_false,
+):
+    """
+    When create a general DataVolume with preallocation set false on DataVolume's spec, preallocation will not be used.
+    It won't take CDI CR into account because it is explicit in the DV.
+    """
+    pvc = registry_dv_with_preallocation_false.pvc
+    assert_preallocation_requested_annotation(pvc=pvc, status="false")
+
+
+@pytest.mark.polarion("CNV-5737")
+@pytest.mark.gating()
```

Ruff (0.15.17)

[warning] Use `@pytest.mark.gating` over `@pytest.mark.gating()`

Remove parentheses

(PT023)